packages/vuepress-theme-odyssey/components/DocsNav.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <nav
+    :class="{
+      'docs-nav': true,
+      [`is-docs-nav-${variant}`]: variant
+    }"
+  >
+    <ul class="docs-nav--list">
+      <li
+        v-for="(item, index) in nav"
+        :key="item.name"
+        :class="{
+          'docs-nav--item': true,
+          'docs-nav--item-with-subnav': item.children
+        }"
+      >
+        <Component
+          :is="item.children ? 'div' : 'fragment'"
+          :class="{
+            'docs-nav--item-content': item.children
+          }"
+        >
+          <DocsLink
+            :id="'nav-subhead-' + index"
```

Comment:
Because we have multiple instances of `DocsNav` on the page, this `id` will **not** [be unique](https://dequeuniversity.com/rules/axe/4.0/duplicate-id-active?application=AxeChrome).

As an example, take a look at the `id` values generated on **Base** and **Updates** once this renders.

Reply:
Great catch. As a quick fix, I have fixed this by concatenating the `variant` to the id. This works fine for our docs site because there are only two variants, but isn't optimal. Given that we're only using this on the docs site, it should be good to go!